demo_nodes_cpp/src/parameters/even_parameters_node.cpp

```diff
@@ -31,6 +30,11 @@ class EvenParameterNode : public rclcpp::Node
   {
     // Force flush of the stdout buffer.
     setvbuf(stdout, NULL, _IONBF, BUFSIZ);
+
+    fprintf(stderr, "This example node shows a parameter callback that rejects\n");
+    fprintf(stderr, "all parameter updates except for those that set an even integer.\n");
+    fprintf(stderr, "Try running 'ros2 param set /even_parameters_node myint 2' to test it.\n");
```

Comment:
Should this be on `stderr`? Seems like "information" to me (rather than a "warning" or "error") so why not just `RCLCPP_INFO`?

Reply:
You are totally correct.  Fixed in b7c8d5d